src/audio/pipeline.c

```diff
@@ -281,6 +280,17 @@ static int pipeline_comp_params(struct comp_dev *current, void *data, int dir)
 	/* send current params to the component */
 	current->params = ppl_data->params->params;
 
+	/* set frames from samplerate/period, but round integer up */
```

Comment:
Can we roll this into https://github.com/thesofproject/sof/issues/1280 and  https://github.com/orgs/thesofproject/teams/sof-developers/discussions/9 . These are related and should be in the same PR for QA.